### PR TITLE
fix some doc inconsistencies and errors

### PR DESCRIPTION
**Pull request by tthsqe12**

### CODE REVIEW

**Comment by thofma** [comment]
Did you find this convention in the developer guide?

**Comment by tthsqe12** [comment]
It is the prevalent mood throughout the docs. Better to have it consistent (and correct).

**Comment by thofma** [comment]
Sure! But we can also add it to the developer guide so that next year we don't change it again.

File: src/flint/fmpq.jl [hunk 1]
```diff
@@ -720,7 +720,7 @@ end
 @doc Markdown.doc"""
     harmonic(n::Int)
 
-Computes the harmonic number $H_n = 1 + 1/2 + 1/3 + \cdots + 1/n$.
+Returns the harmonic number $H_n = 1 + 1/2 + 1/3 + \cdots + 1/n$.
```
Comment (rfourquet) [inline src/flint/fmpq.jl:723]:
```suggestion
Return the harmonic number $H_n = 1 + 1/2 + 1/3 + \cdots + 1/n$.
```
;-)

**Pull request merged**